src/app/framework-modules/elements/ct-field-generator/component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, Output, EventEmitter} from '@angular/core';
+import {FormBuilder, FormGroup, Validators, FormControl} from '@angular/forms';

```

Comment:
Formatting nitpick.  You have a space after `{` and a space before `}` on the neighboring lines, but not here.  Reformat/lint this line for consistency?